### ISSUE TITLE
Can we stay logged in forever?

### ISSUE THREAD

**Issue by b3nsh4** [Harishwarrior/jio_balance#1]
As we see in Jio official app, can we have a way to stay the session active as long as we uninstall the app itself? In Official application that what is making everyone to use the app. Else we need login at each time. 

Indeed this *Jio_balance* app is useful software which strips the bloatware indeed.

**Comment by b3nsh4** [comment]
Just saw your To-do list. So closing!

**Comment by Harishwarrior** [comment]
I found a hack to stay logged in. Didn't tested it yet.

**Comment by b3nsh4** [comment]
> I found a hack to stay logged in. Didn't tested it yet.

I believe it's related to `JSESSION` where it has an expiration time of roughly ~10hrs which I saw from my browser after seeing few requests and responses. May be the official app has a different session property which won't expire?

**Comment by Harishwarrior** [comment]
> > I found a hack to stay logged in. Didn't tested it yet.
> 
> I believe it's related to `JSESSION` where it has an expiration time of roughly ~10hrs which I saw from my browser after seeing few requests and responses. May be the official app has a different session property which won't expire?

Exactly, we must find out what type of session the app is using for this feature to be implemented. You can check Airtel Lite app for reference.

**Comment by b3nsh4** [comment]
The key is to identify _how the server understand that it's the official Jio app_ . That is what makes the difference between a browser and the app. I'm not sure if it is just header only. Will look into it! :+1:
